Description: Lintian warnings due to external HTTP calls
 Removed external image references to prevent possible privacy concerns
 as highlighted by Lintian.
Author: Anuradha Weeraman <anuradha@debian.org>
Last-Update: 2024-06-02
Forwarded: not-needed

--- a/README.rst
+++ b/README.rst
@@ -2,29 +2,8 @@
 Srain
 =====
 
-|ci-status| |docs-status| |channel|
-
-.. |ci-status| image:: https://github.com/SrainApp/srain/workflows/build/badge.svg?branch=master
-   :target: https://github.com/SrainApp/srain/actions
-
-.. |docs-status| image:: https://github.com/SrainApp/srain/actions/workflows/pages.yml/badge.svg
-    :target: https://srain.silverrainz.me/
-
-.. |channel| image:: https://img.shields.io/badge/channel-%23srain-brightgreen.svg
-   :target: ircs://irc.libera.chat/#srain
-
 Modern IRC client written in GTK.
 
-.. image:: https://srain.silverrainz.me/_images/main.png
-
-Packaging Status
-================
-
-|pkgstat|
-
-.. |pkgstat| image:: https://repology.org/badge/vertical-allrepos/srain.svg?columns=3
-   :target: https://repology.org/project/srain/versions
-
 Features
 ========
 
@@ -49,21 +28,6 @@ Features
   - Regex based message render and filter mechanisms
   - IPC based plugin system (TODO)
 
-.. _IRCv3: https://ircv3.net/
-
-Need Help?
-==========
-
-`Frequently Asked Questions`_ may be helpful.
-
-Feel free to contact us if you have any question about Srain.
-
-- Github: file an issue `Here <https://github.com/SrainApp/srain/issues>`_
-- IRC Channel: I'm no longer active on IRC, but you can still ask other users
-  for help on `#srain <ircs://irc.libera.chat/srain>`_ @ Libera Chat
-
-.. _Frequently Asked Questions: https://srain.silverrainz.me/faq.html
-
 License
 =======
 
